person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  def iinitialize(name, age = 'unknown', parent_permission = 'true')
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
```

Comment:
- [ ] The id should be generated randomly, so it would be present in the initialize like `@id = Random.rand(1..1000)`. Please kindly fix that 😉😉